src/components/UserDetailedSymptomsCard/UserDetailedSymptomsCard.js

```diff
@@ -1,23 +1,29 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
 import Card from "react-bootstrap/Card";
 import ListGroup from 'react-bootstrap/ListGroup';
 import {ListGroupItem} from "react-bootstrap";
 import ReportsAPI from "../../services/ReportsAPI";
 
 const UserDetailedSymptomsCard = ({report, vitalSigns}) => {
+    const [reportData, setReportData] = useState([]);
```

Comment:
Shouldn't it be: const [reportData, setReportData] = useState({});?

Reply:
You're right! Done